### DISCUSSION TITLE
java.lang.NoClassDefFoundError: net/minecraft/text/Text

### DISCUSSION THREAD

**Discussion by ExquisiteCore** [FabricMC/fabric-api#3320] [Mod Dev Support]
### Description

I don't see this error in my debugging environment within IDEA, but when I export the MOD to the server, it throws an error.
![image](https://github.com/FabricMC/fabric/assets/59426890/9e5222f1-8982-4a2a-9b71-c9da00e8d968)

**Comment by takayamaekawa** [comment]
Me too.
I can't solve this error:`java.lang.NoClassDefFoundError: net/minecraft/text/Text`.

**Reply by Linguardium** [reply]
if its showing in your production environment, its because you exported your mod wrong. use the gradle "build" task and find your built production jar in the "build/libs/" folder. be sure you dont use the devlibs or dev jar

**Reply by takayamaekawa** [reply]
I already know that, of course.
